Breadcrumbs.py

```diff
@@ -108,4 +119,193 @@ def get_points_by_row(row):
               breadcrumbs[index_of_breadcrumb] = breadcrumbs[index_of_breadcrumb][0:length_of_trim]
           break
       number_of_characters_left -= current_length
-    view.set_status('breadcrumbs',separator.join(breadcrumbs+['']))
+
+  return breadcrumbs
+
+
+def copy(view, text):
+  sublime.set_clipboard(text)
+  view.hide_popup()
+  sublime.status_message('Breadcrumbs copied to clipboard')
+
+
+class BreadcrumbsCommand(sublime_plugin.EventListener):
+
+  def on_selection_modified(self, view):
+    view.erase_status('breadcrumbs')
+
+    settings = sublime.load_settings('Breadcrumbs.sublime-settings')
+    default_statusbar_enabled = settings.get('show_breadcrumbs_in_statusbar', True)
+    statusbar_enabled = view.settings().get('show_breadcrumbs_in_statusbar', default_statusbar_enabled)
+
+    if statusbar_enabled:
+      separator = settings.get('breadcrumbs_separator', u' › ')
+      current_row = view.rowcol(view.sel()[0].b)[0]
+      breadcrumbs = make_breadcrumbs(view, current_row)
+
+      if breadcrumbs is not None and len(breadcrumbs) > 0:
+        view.set_status('breadcrumbs', separator.join(make_breadcrumbs(view, current_row, shorten=True)))
```

Comment:
Since evaluation of `make_breadcrumbs` is quite heavy, I would suggest to call it only once if it is possible. Currently, you call it once in line 144, and here again with `shorten=True`. Is there something preventing us from calling it with `shorten=True`  in line 144 and reuse `breadcrumbs` variable here?